src/test/java/com/ibm/cloud/sdk/core/test/security/ConfigBasedAuthenticatorFactoryTest.java

```diff
@@ -217,6 +231,15 @@ public void testFileCredentialsService4() {
     assertEquals(auth.authenticationType(), Authenticator.AUTHTYPE_NOAUTH);
   }
 
+  @Test
```

Comment:
All I did with this code block is I moved it to be right after the test re: "service4".